<launch>
    <arg name="init_pcd_file" default="$(find HookCollisionDet)/data/test/1-load.pcd"/> <!-- path to init pcd file -->
    <arg name="test_pcd_files_dir" default="$(find HookCollisionDet)/data/test/"/> <!-- path to test pcd files fold-->

    <node pkg="HookCollisionDet" type="HookCollisionDet" name="hook_det" output="screen">
        <param name="init_pcd_file" value="$(arg init_pcd_file)"/>
        <param name="test_pcd_files_dir" value="$(arg test_pcd_files_dir)"/>
        <param name="rope_len" value = "30.0"/>
    </node >
</launch>
